mingw-w64-mysql_client/PKGBUILD

```
@@ -0,0 +1,40 @@
+# Maintainer: taozuhong <taozuhong@gmail.com>
+
+_realname=mysql
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}-client"
+pkgver=8.0.25
+pkgrel=1
+pkgdesc="MySQL client libraries (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64')
+url="https://dev.mysql.com/doc/refman/8.0/en/"
+license=("GPL-2.0-only with Universal-FOSS-exception-1.0")
+makedepends=("${MINGW_PACKAGE_PREFIX}-cmake"
+             "${MINGW_PACKAGE_PREFIX}-boost")
+depends=("${MINGW_PACKAGE_PREFIX}-libevent"
+         "${MINGW_PACKAGE_PREFIX}-zstd"
+         "${MINGW_PACKAGE_PREFIX}-openssl")
+source=("https://cdn.mysql.com/Downloads/MySQL-8.0/mysql-${pkgver}.tar.gz")
+sha256sums=('c16aa9cf621bc028efba2bb11f3c36a323b125fa0d108ff92fab60e46309206e')
+            
+prepare() {
+  cd ${srcdir}/${_realname}-${pkgver}
+}
+
+build() {
+  rm -rf "${srcdir}/build-${MINGW_CHOST}"
+  mkdir "${srcdir}/build-${MINGW_CHOST}"
+  cd "${srcdir}/build-${MINGW_CHOST}"
+  
+  MSYS2_ARG_CONV_EXCL=- cmake ../${_realname}-${pkgver} \
+    -G"MSYS Makefiles" \
+    -DCMAKE_INSTALL_PREFIX=$MINGW_PREFIX \
+    -DWITH_SSL=system \
+    -DFORCE_INSOURCE_BUILD=1
+}
+
+package() {
+  cd "${srcdir}/build-${MINGW_CHOST}"
+  
+  make install DESTDIR="${pkgdir}"
```

```suggestion
  DESTDIR="${pkgdir}" cmake --build . --target install
```

It might be better to use cmake to handle the `make`ing in case the generator is changed or overridden by a future change